### PR TITLE
Feature/i18n update

### PR DESCRIPTION
**Pull request by emclaug2**
Changes proposed in this Pull Request:
- Update i18n example to use ngx-translate.
- Enforce translations with typescript.
- Add linting and prettier
- Clean up SCSS inline rules
- Rewrite unit tests 
- Add german, french, arabic, chinese, english, spanish, portuguese examples 
- Update to ng9

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "internationlization": {
+    "pxb-i18n": {
```
Comment (joebochill) [inline angular.json:6]: why are we changing the project name?

Comment (emclaug2) [reply angular.json]: I reverted the project name back to `internationalization`, I think I was playing around with something using another project's configuration then forgot to swap it back.

**Comment by emclaug2** [comment]
> It's funny....the first time we wrote this example, we used ngx-translate and then switched the implementation to match how we were doing it in react. Now we've come full circle

Maybe once @angular/localize supports more than 1 localization per build we'll change it again ;)

**Comment by joebochill** [comment]
> > It's funny....the first time we wrote this example, we used ngx-translate and then switched the implementation to match how we were doing it in react. Now we've come full circle
> 
> Maybe once @angular/localize supports more than 1 localization per build we'll change it again ;)

I'm sure this won't be the last time

File: angular.json [hunk 2]
```diff
@@ -92,37 +91,12 @@
...
-              "devServerTarget": "internationlization:serve:production"
-            }
-          }
         }
       }
     }},
-  "defaultProject": "internationlization"
-}
\ No newline at end of file
+  "defaultProject": "internationalization"
```
Comment (joebochill) [inline angular.json:101]: I approve of this name change now :)

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,31 +1,97 @@
...
+                <mat-toolbar-row style="height: 4px;">
+                    <small>PX {{ 'BLUE' | translate }}</small>
+                </mat-toolbar-row>
+                <mat-toolbar-row>
+                    <small style="margin-top: -24px;" id="i18n">{{ 'I18N' | translate }}</small>
+                </mat-toolbar-row>
+            </mat-toolbar>
+            <mat-divider></mat-divider>
+            <div class="nav-wrapper">
+                <a mat-list-item (click)="toggleMenu()">
```
Comment (joebochill) [inline src/app/app.component.html:17]: Would be nice to use our infoListItem component here...if I published a beta of the angular-components, would it make sense to integrate it now?

Comment (emclaug2) [reply src/app/app.component.html:17]: I'll update the example to use InfoListItem component.

Comment (joebochill) [reply src/app/app.component.html:17]: @pxblue/angular-components@2.0.0-beta.3 is now available

**Pull request merged**